redis_test.go

```diff
@@ -737,6 +827,65 @@ func testComputeTimeout(t *testing.T) {
 	}
 }
 
+func createUser(hostname string, port int, redisTls bool, caCrt, adminuser, adminpassword, username, password, aclRule string) (err error) {
+	pem, err := base64.StdEncoding.DecodeString(caCrt)
+	if err != nil {
+		return fmt.Errorf("error decoding CaCrt")
+	}
```

Comment:
We removed requirement for base64 encoding of the PEM file, so we can do the same here.

Reply:
@austingebauer Thank you so much for the suggestions! 

Hm, I am curious about this one. So, when I removed this portion of the code, since we did delete the field for base64prem. I am getting an error that I am unable to parse the certificate. Would we still need this since we are passing the certificate encoded with base64, therefore needs to be encoded before appending?

I did also try appending the CACert instead, but I get the same error.

Reply:
I guess this would be a better question for the suggestion above

Reply:
I think we'll want to remove `base64.StdEncoding.DecodeString(c.CACert)` in any case, since we're expecting that the CA cert is passed without being base64 encoded. I would double check that the CA cert you're supplying to the test isn't base64 encoded.

Reply:
Got it! 👍